pkg/util/jsonnet/jsonnet.go

```diff
@@ -0,0 +1,40 @@
+// JSONNET Decoder
+package jsonnet
+
+import (
+	"bytes"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/golang/glog"
+	utilexec "k8s.io/kubernetes/pkg/util/exec"
+)
+
+const cmdJsonnet string = "jsonnet"
+
+// Run converts a single JSONNET document into a JSON document
+func Run(args ...string) ([]byte, error) {
+	glog.V(4).Infof("running jsonnet %v", args)
+	return utilexec.New().Command(cmdJsonnet, args...).CombinedOutput()
```

Comment:
Yes, I also want to apply a jsonnet go library. But go-jsonnet is still in proress and unable to convert a file into json document.